### PR TITLE
Make image file upload test wait for image to load before trying to read it

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Possibly helps with https://github.com/dotnet/aspnetcore/issues/25929

I don't know whether this is actually going to make a difference. Running locally this test *always* fails for me on Blazor Server, and the change in this PR fixes that. Running in CI though, it virtually always passes on Server and occasionally fails on WebAssembly.

Since this is the only thing I can guess might be going wrong, let's try fixing that.

### CODE REVIEW

File: src/Components/test/E2ETest/Tests/InputFileTest.cs [hunk 1]
```diff
@@ -135,6 +135,8 @@ public void CanUploadMultipleFiles()
         public void CanUploadAndConvertImageFile()
         {
             var sourceImageId = "image-source";
+            var imageStatus = Browser.Exists(By.Id("image-status"));
+            Browser.Equal("ready", () => imageStatus.Text);
```
[the remark this answers is not in the document]
Comment (SteveSandersonMS) [reply src/Components/test/E2ETest/Tests/InputFileTest.cs:139]: Yes

File: src/Components/test/E2ETest/Tests/VirtualizationTest.cs [hunk 2]
```diff
@@ -168,7 +168,6 @@ public void RerendersWhenItemSizeShrinks_Async()
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/25929")]
```
Comment (SteveSandersonMS) [inline src/Components/test/E2ETest/Tests/VirtualizationTest.cs:171]: This was fixed in my other PR - I should have removed it there.

**Review by TanayParikh — APPROVED** [review]
LGTM

**Review by mkArtakMSFT — APPROVED** [review]

**Review by pranavkm — APPROVED** [review]

**Review by javiercn — APPROVED** [review]

File: src/Components/test/E2ETest/Tests/VirtualizationTest.cs [hunk 3]
```diff
@@ -169,7 +169,6 @@ public void RerendersWhenItemSizeShrinks_Async()
         }
 
         [Fact]
```
Comment (SteveSandersonMS) [inline src/Components/test/E2ETest/Tests/VirtualizationTest.cs:171]:
```suggestion
        [Fact]
        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/25929")]
```

**Pull request merged**